sdk/identity/identity/src/credentials/azurePipelinesCredential.ts

```diff
@@ -149,23 +163,14 @@ export class AzurePipelinesCredential implements TokenCredential {
    * @throws CredentialUnavailableError
    */
   private ensurePipelinesSystemVars(): void {
-    if (
-      process.env.SYSTEM_TEAMFOUNDATIONCOLLECTIONURI &&
-      process.env.SYSTEM_TEAMPROJECTID &&
-      process.env.SYSTEM_PLANID &&
-      process.env.SYSTEM_JOBID &&
-      process.env.SYSTEM_ACCESSTOKEN
-    ) {
+    if (process.env.SYSTEM_OIDCREQUESTURI && process.env.SYSTEM_ACCESSTOKEN) {
```

Comment:
Since the system access token is passed into the constructor, we no longer need the check for SYSTEM_ACCESSTOKEN in the environment, right?

Reply:
I believe we should still add this check so that we can verify if the pipeline config is set up correctly.

Reply:
Hmm but if the user is passing this to the constructor, how do we know it's named SYSTEM_ACCESSTOKEN and not MAORS_SUPER_SECRET_ACCESS_TOKEN? That was the concern that led us to taking it in as a constructor argument right?

Reply:
Yes I removed that check now. And just added in the reference docs, TS guide and examples section to configure this.